dmoj/cptbox/compiler_isolate.py

```diff
@@ -13,6 +16,25 @@
 UTIME_OMIT = (1 << 30) - 2
 
 
+def syscall_simulator(
+    simulator: Callable[['CompilerIsolateTracer', Debugger], None]
+) -> Callable[['CompilerIsolateTracer', Debugger], None]:
+    def inner(tracer: 'CompilerIsolateTracer', debugger: Debugger) -> None:
```

Comment:
Should we `@functools.wraps(simulator)`?

Reply:
Unfamiliar with that function. I can look into it (or blindly follow your suggestion).

Reply:
Added.